CHANGELOG.md

```diff
@@ -10,7 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat: Add keypair pub/priv example
 - feat: use policy & role modules in complete example
 - feat: replace lambda stand-alone resources with boldlink lambda module in complete example
+- feat: update module used to create a vpc in the complete example
+
```

Comment:
Add note `feat: use lambda module in complete example lambda resources"

Reply:
@patrickmukumbu we have that already captured in CHANGELOG.md

Reply:
Sorry I hadn't seen this @ndegwajohn